src/apiHandle.js

```diff
@@ -0,0 +1,27 @@
+const getData = async (url) => {
+  const scores = await fetch(url)
+    .then((res) => res.json())
+    .then((resData) => resData.data.memes)
+    .catch((err) => err);
+  return scores;
+};
+
+const likeMeme = async (api, memeId) => {
+  await fetch(api, {
+    method: 'POST',
+    headers: { 'Content-type': 'application/json; charset=UTF-8' },
+    body: JSON.stringify({
+      item_id: memeId,
+    }),
+  });
+};
+
+const getLikes = async (api) => {
+  const resp = await fetch(api)
+    .then((res) => res.json())
+    .then((resData) => resData)
+    .catch((err) => err);
+  return resp;
```

Comment:
- [ ] The likes API is not working correctly and not storing the likes after refreshing the page.

![llllll](https://user-images.githubusercontent.com/61562626/143677751-f3aadca3-cd36-4c9f-8638-9d057dd88db9.jpeg)

Reply:
- [ ] Still, this issue persists, after refreshing the browser or opening the application the `number of likes` is not showing. Please make sure the likes are fetching correctly on page load.

![Screenshot from 2021-11-27 18-18-47](https://user-images.githubusercontent.com/22434660/143682009-6ce0430e-db2e-4241-996c-e4939c2fc1f7.png)

Reply:
Hi Depeshtwis,
Thanks for your feedback. It looks like the likes are taking a bit longer to update, but they are definitely working. I have recorded a screencast for you to see: https://share.getcloudapp.com/llukedkB.
Please let us know if there's anything else. 
Thanks